app/models/ring.rb

```diff
@@ -1,19 +1,18 @@
 class Ring
   def initialize(redirection_to_link)
-    @oldest_redirection = Redirection.order(created_at: :desc).last
-    @newest_redirection = Redirection.order(created_at: :desc).first
     @redirection_to_link = redirection_to_link
+    @from_redirection = Redirection.first
+    @to_redirection = from_redirection.next
```

Comment:
What about setting these as methods?

I'm not sure if someone told me it was a good idea, but I'm only in the habit of doing param-to-ivar work in the constructor and putting everything else in a method. Interested in your feedback.